container/features/src/main/resources/features.xml

```diff
@@ -1216,8 +1216,12 @@
     </feature>
     <feature name="opennms-topology-runtime-application" version="${project.version}" description="OpenNMS :: Features :: Topology :: Features :: Application">
         <details>Generated using Pax-Construct</details>
+        <!-- TODO: patrick , mvr fix dependency problems -->
```

Comment:
@mvrueden it seems I can't get the dependencies right here. Maybe you want to take a look?

Reply:
I fixed that with various other things.